client/test/components/conference_room/provider/EventProvider.spec.jsx

```diff
@@ -106,4 +111,82 @@ describe('<EventProvider />', () => {
       });
     });
   });
+
+  describe('confirming event', () => {
+    const wrapper = mount(<EventProvider conferenceRoom={conferenceRoom} component={DummyComponent} />);
+    const componentWrapper = wrapper.find(DummyComponent);
+
+    before(() => {
+      sinon.stub(EventSource, 'confirm');
+    });
+
+    after(() => {
+      EventSource.confirm.restore();
+    });
+
+    context('with successful request', () => {
+      before(() => {
+        wrapper.setState({ currentEvent: Event.build({ confirmed: false }) });
+        EventSource.confirm.resolves();
+        componentWrapper.find('button#confirm').simulate('click');
+      });
+
+      it('toggles current event\'s confirmation', () => {
+        expect(componentWrapper.prop('currentEvent').confirmed).to.be.true();
```

Comment:
`subject = componentWrapper.prop('currentEvent').confirmed`
and
`expect(subject).to.be.true()`